client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-expanded-content/boost-site-performance.tsx

```diff
@@ -39,7 +41,7 @@ export default function BoostSitePerformance( {
 
 	const href = `${ siteUrlWithScheme }/wp-admin/admin.php?page=jetpack-boost`;
 
-	return (
+	return isBoostEnabled ? (
```

Comment:
I think we can remove these changes and handle this in the `SiteExpandedContent` component by doing the below.

Instead of 

```
const defaultColumns = [ 'stats', 'boost', 'backup', 'monitor' ];
```

We can do so that the columns are controlled in one centralized place

```
import { siteColumns } from '../utils';
...
...
const defaultColumns = siteColumns.map( ( { key } ) => key );
```

Reply:
By doing this, when we enable the Boost column, it'll be visible in both places. WDYT?